utils/data-processor.js

```diff
@@ -169,13 +169,13 @@ class DataProcessor {
   }
 
   /*
-   * Recalculates isReliable for given row
+   * Recalculates reliable for given row
    * @param{Row} row - The row to update
    * @param{boolean} wasCoded - Flag indicating if the estimate value has been coded
    */
-  recalculateIsReliable(row, wasCoded) {
+  recalculateReliable(row, wasCoded) {
     // top- or bottom-coded values are not reliable
-    row.isReliable = wasCoded ? false : executeFormula(this.data, row.variable, 'isReliable');
+    row.reliable = wasCoded ? false : executeFormula(this.data, row.variable, 'reliable');
```

Comment:
Is there a particular reason you went through the trouble of renaming the existing instances of `isReliable` to be `reliable`? I personally like the `is` naming convention for booleans but I know it's not something we've ever talked about as a group.

Reply:
Changed it to be consistent with `changeReliable` `previousReliable`, etc.  Agree that `is` is generally what we should be using for booleans. I would rather change all the variables to be like `changeIsReliable`. But the key here is consistency